### DISCUSSION TITLE
How does Timesketch work with Neo4j graph database?

### DISCUSSION THREAD

**Discussion by bronzeMe** [google/timesketch#1739] [Q&A, quick solutions, support] [answered]
Hi, everyone, it seems timesketch supports using neo4j as its backend graph database.
However, I did not find some tutorials about how do I integrate timesketch with neo4j,
Could someone give some hints? thanks  a lot.

**Comment by kiddinn** [comment, marked as answer]
Graphing is now supported, but we are not using neo4js anymore.

You can see the new graphing functionality in the "graph" section of Timesketch, see:

![graph_picture](https://user-images.githubusercontent.com/8701947/121772521-5bf74800-cb65-11eb-8b21-90f10d4e9f7c.png)

Just click the `Graph` button in the interface and then you can click one of the supported plugins.

We don't have ATM any documentation on how to add new plugins, that will be coming soon, hopefully, but you can take a look at other plugins and then just copy and adjust them to your needs if you want to generate a new one. Plugins are [here](https://github.com/google/timesketch/tree/master/timesketch/lib/graphs).

**Comment by bronzeMe** [comment]
Thank you so much, I will try it !
